tock/tock/templates/_navigation.html

```diff
@@ -1,49 +1,52 @@
-<nav role="navigation" class="usa-nav">
```

Comment:
there's debate about whether having roles like this on HTML5 native elements is actually harmful and confusing to screen readers, so I took this out.